pom.xml

```diff
@@ -31,12 +31,6 @@
             <version>3.2.4</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.maven</groupId>
```

Comment:
pls remove the apache.httpclient dependency as well.

Reply:
yup that one is gone too :)